src/utils/index.ts

```diff
@@ -1,11 +1,7 @@
-// disabled because cycle is created by outside files that import files from
-// this folder - it wouldn't make sense to separate them there
-// and break imports concept just to remove eslint error here
-
 export { default as abbreviations } from './abbreviations';
 export { default as addressUtils } from './address';
 export { default as AllowancesCache } from './allowancesCache';
-export { default as buildRainbowUrl } from './buildRainbowUrl';
+export * from './buildRainbowUrl';
```

Comment:
I think we don't need to re-export from utils. Not sure about its performance implications or just general bundle implications, but if we can abandon the nicer `from '@/utils';` in favor of `from '@/utils/buildRainbowUrl';`. Right now TypeScript has auto-import features, VSCode had that before TypeScript had it and IntelliJ had that ages ago.

Reply:
+1 fwiw this is in notion already as a best practice, but we can talk more about it!